backoffice/src/Controllers/EmailController.cs

```diff
@@ -50,16 +50,30 @@ public async Task<IActionResult> Compose(Guid id)
     [HttpGet]
     public async Task<IActionResult> SendEmail(Guid id)
     {
-        await _service.SendEmailAsync(id);
+        try
+        {
+            await _service.SendEmailAsync(id);
+            TempData["SuccessMessage"] = "Email sent successfully";
+        }
+        catch (Exception e)
+        {
+            _logger.LogError(e, "Email not found");
+            TempData["ErrorMessage"] = "Email not found";
+        }
         
-        //Add feedback (banner or something) to show sent. 
         return RedirectToAction(nameof(Index));
     }
 
     [HttpPost]
     public async Task<IActionResult> SaveEmail(EmailForm form)
     {
-        Email email; 
+        Email email;
+
+        if (!ModelState.IsValid)
+        {
+            TempData["ErrorMessage"] = "One or more form fields are incorrect or missing";

```

Comment:
Something more generic and shorter would be fine. Something like 

"An error occurred while saving the email"